### DISCUSSION TITLE
Ability to start jobrunr and then terminate automatically

### DISCUSSION THREAD

**Discussion by goafabric** [jobrunr/jobrunr#833] [Q&A]
Hello,
I've got a rather simple requirement for jobrunr.

For stuff like importing CSV Files, i would look jobrunr to start,
execute all the jobs.
And then terminate after all the jobs have finished successfully.

start application -> execute all jobs -> all jobs finished -> terminate application

But i haven't found a way to do so.
You can configure jobrunr to either start all jobs and block.
Or not start at all.

So is there a simple option for this ?

Or do i have to use one of the listeners available ? And then how ?

Thx in advance

**Comment by rdehuyss** [comment]
JobRunr was not designed for that, to be honest. It's was designed not only for one-off/enqueued jobs but also for recurring and scheduled jobs, hence the need to run forever. 

In the Pro version we're looking at adding jobs in Flyway and Liquibase Migrations, e.g. you could run jobs while deploying but this is a planned feature for the moment.

**Comment by goafabric** [comment]
ok thx again
